Tools/TestWebKitAPI/Tests/WebKitGLib/TestDownloads.cpp

```diff
@@ -816,6 +816,7 @@ static void testDownloadEphemeralContext(Test* test, gconstpointer)
         GUniquePtr<char> destination(g_build_filename(Test::dataDirectory(), suggestedFilename, nullptr));
         GUniquePtr<char> destinationURI(g_filename_to_uri(destination.get(), nullptr, nullptr));
         webkit_download_set_destination(download, destinationURI.get());
+        return true;
```

Comment:
Careful, this needs to be `return TRUE` since it's a `gboolean` (`int`, not a `bool`)